common/thorhelper/thorsoapcall.cpp

```diff
@@ -50,6 +50,41 @@ using roxiemem::OwnedRoxieString;
 #define CONNECTION "Connection"
 
 unsigned soapTraceLevel = 1;
+StringBuffer soapSepString;
```

Comment:
looks like this should now be a static

Reply:
I didn't add a get method and the string is used in a few places so I left it as global.

Reply:
ok, yes